axelor-tool/src/main/java/com/axelor/apps/tool/StringTool.java

```diff
@@ -268,19 +269,21 @@ public static String capitalizeFirstLetter(String s) {
    * @return
    */
   public static String getIdListString(Collection<? extends Model> collection) {
-    List<Long> idList = new ArrayList<>();
-    String idString;
+    StringBuilder idStringBuilder = new StringBuilder();
     if (CollectionUtils.isEmpty(collection)) {
-      idString = "0";
+      idStringBuilder.append("0");
     } else {
       for (Model item : collection) {
```

Comment:
```suggestion
      return "0";
    }
    
    for (Model item : collection) {
       ...
```

We can reduce cognitive complexity and return immediatly the value. As a consequence, we don't need an `else` block below.